docs/markdown/Tutorial.md

```diff
@@ -29,10 +29,13 @@ example. First we create a file `main.c` which holds the source. It
 looks like this.
 
 ```c
-#include<stdio.h>
+#include <stdio.h>
 
+//
+// main is where all program execution starts
+//
 int main(int argc, char **argv) {
-  printf("Hello there.\n");
+  puts("Hello there.");
```

Comment:
This may be confusing to newcomers. Most people know about `printf`(the canonical helloworld uses printf, after all) but knowledge about `puts` is less common. This is not even a performance improvement, as compilers will do this transformation automatically:

https://gcc.godbolt.org/z/Kxcqhb

Reply:
I was actually more focused on making the snippet more secure but I well happy to change it back to `printf`.